plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/editors/entity/properties/ObjectPropertiesTabFolderSwitchCommandHandler.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ui.editors.entity.properties;
+
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.ui.IEditorPart;
+import org.eclipse.ui.handlers.HandlerUtil;
+import org.eclipse.ui.part.MultiPageEditorPart;
+import org.jkiss.dbeaver.ui.controls.folders.TabbedFolderInfo;
+import org.jkiss.utils.ArrayUtils;
+
+import java.util.Map;
+
+public class ObjectPropertiesTabFolderSwitchCommandHandler extends AbstractHandler {
+    
+    public static final String NEXT_PAGE_COMMAND_ID = "org.jkiss.dbeaver.entity.propsTab.nextPage";
+    public static final String PREV_PAGE_COMMAND_ID = "org.jkiss.dbeaver.entity.propsTab.prevPage";
+    
+    private static final Map<String, Integer> pageIndexDeltaByCommandId = Map.of(
+        NEXT_PAGE_COMMAND_ID, 1,
+        PREV_PAGE_COMMAND_ID, -1
+    );
+    
+    @Override
+    public Object execute(ExecutionEvent event) throws ExecutionException {
+        Integer indexDelta = pageIndexDeltaByCommandId.get(event.getCommand().getId());
+        IEditorPart editor = HandlerUtil.getActiveEditor(event);
+        if (indexDelta != null && editor instanceof MultiPageEditorPart) {
+            MultiPageEditorPart pagedEditor = (MultiPageEditorPart) editor;
+            Object currentPage = pagedEditor.getSelectedPage();
+            if (currentPage instanceof ObjectPropertiesEditor) {
+                ObjectPropertiesEditor propsEditor = (ObjectPropertiesEditor) currentPage;
+                TabbedFolderInfo[] foldersInfo = propsEditor.collectFolders(propsEditor);
+                String activeFolderId = propsEditor.getActiveFolderId();
+                int activeFolderIndex = ArrayUtils.indexOf(foldersInfo, f -> f.getId().equals(activeFolderId));
+                int targetFolderIndex = activeFolderIndex + indexDelta;
+                if (targetFolderIndex >= 0 && targetFolderIndex < foldersInfo.length) {
+                    propsEditor.switchFolder(foldersInfo[targetFolderIndex].getId());
+                }
```

Comment:
WDYT of wrapping the index around? It would be useful IMO.

The "wrap-around" function can be expressed as so:
```java
public static int wrapAround(int index, int max) {
    return (index % max + max) % max;
}
```

It is proved to be working.

Reply:
I can do it, but it was not working like that before. Not sure, which is a good behaviour from the point of accessibility, but ok.

Why don't just `(index + max) % max` ?

Reply:
Never mind

Reply:
Done
I don't want to make it an utility method since here I know the the range of possible values, but in utility method I'll need to work with any value and match it to the function domain. I don't need it somewhere else, so it's unnecessary complexity.